packages/web-components/src/components/cbp-toast/cbp-toast.scss

```diff
@@ -0,0 +1,149 @@
+/**
+ * @Prop   --cbp-toast-background: var(--cbp-color-info-dark);
+ * @Prop   --cbp-toast-background-dark: var(--cbp-color-info-lighter);
+ * @Prop   --cbp-toast-color: var(--cbp-color-text-lighter);
+ * @Prop   --cbp-toast-color-dark: var(--cbp-color-text-darker);
+ * @Prop   --cbp-toast-color-content: var(--cbp-color-text-lighter);
+ * @Prop   --cbp-toast-color-content-dark: var(--cbp-color-text-darkest);
+ * @Prop   --cbp-toast-sidebar: var(--cbp-color-info-darker);
+ * @Prop   --cbp-toast-sidebardark: var(--cbp-color-info-light);
+ * @Prop   --cbp-toast-sidebar-icon: var(--cbp-color-text-light);
+ * @Prop   --cbp-toast-sidebar-icon-dark: var(--cbp-color-text-darker);
+ */
+ :root{
+ 
+  --cbp-toast-background: var(--cbp-color-info-dark);
+  --cbp-toast-background-dark: var(--cbp-color-info-lighter);
+  --cbp-toast-color: var(--cbp-color-text-lighter);
+  --cbp-toast-color-dark: var(--cbp-color-text-darker);
+  --cbp-toast-color-content: var(--cbp-color-text-lighter);
+  --cbp-toast-color-content-dark: var(--cbp-color-text-darkest);
+  --cbp-toast-sidebar: var(--cbp-color-info-darker);
+  --cbp-toast-sidebardark: var(--cbp-color-info-light);
+  --cbp-toast-sidebar-icon: var(--cbp-color-text-light);
+  --cbp-toast-sidebar-icon-dark: var(--cbp-color-text-darker);
+
+}
+
+[data-cbp-theme=light] cbp-toast[context*=dark]:not([context=light-always]),
+[data-cbp-theme=dark] cbp-toast:not([context=dark-inverts]):not([context=light-always]) {
+
+  --cbp-toast-background: var(--cbp-toast-background-dark);
+  --cbp-toast-color: var(--cbp-toast-color-dark);
+  --cbp-toast-color-content: var(--cbp-toast-color-content-dark);
+  --cbp-toast-sidebar: var(--cbp-toast-sidebar-dark);
+  --cbp-toast-sidebar-icon: var(--cbp-toast-sidebar-icon-dark);
+
+}
+
+
+@keyframes show {
+  0%{
+    opacity: 1;
+    transform: translateX(100%);
+  }
+
+}
+
+@keyframes dismiss {
+  99%{
+    transform: translateX(200%);
+  }
+  100%{
+    visibility: hidden;
+  }
+}
+
+cbp-toast {
+  display: flex;
+  margin-left: auto;
+  margin-bottom: var(--cbp-space-6x);
+  width: max-content;
+  max-width: 50%;
```

Comment:
This will not scale well on mobile. You'd want it to be allowed to take most of the width on a small screen. Maybe think about using min() here or on the width itself. See https://www.smashingmagazine.com/2024/10/css-min-all-the-things/